### PR TITLE
Rename source doc for README, first step for #12

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Output geojson from latest commit looks like this:

image

**Comment by Robinlovelace** [comment]
And when the disaggregation key is set to car it's like this, this is working as intended :+1: 

image

File: README.qmd [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-output: github_document
+format: gfm
 ---
 
+<!-- Note: do not edit the README.md file directly. -->
+<!-- Edit README.qmd and rebuild with the following command (install quarto first) -->
+
+<!-- quarto render README.qmd -->
+
 ```{r eval=FALSE, echo=FALSE}
```
Comment (dabreegster) [inline README.qmd:10]: We can delete this bit now

**Review by dabreegster — COMMENTED** [review]
When I rendered, I got a diff at the top:

```
+---
+format: gfm
+pagetitle: readme
+toc-title: Table of contents
+---
+
```

Any idea why?

**Comment by dabreegster** [comment]
So to run quarto on my end, I had to `cargo install --git https://github.com/dabreegster/odjitter`. The problem is if I'm trying to regenerate docs based on my current modified version of odjitter, I have to manually change all the commands to `./target/debug/odjitter`. I wonder if we could use variables (either quarto ones or shell variables that're persisted across the multiple shell fragments) to make this easier?

**Comment by Robinlovelace** [comment]
> I wonder if we could use variables (either quarto ones or shell variables that're persisted across the multiple shell fragments) to make this easier?

Not sure. I think searching and replacing odjitter with ./target/debug/odjitter would be a quick fix for tests or we could just replace it for now while it's in development and agree to switch back to `odjitter` when the API stabilises. Sound reasonable? Advantage of this 'literate programming' approach where the outputs of the docs auto update.

**Comment by Robinlovelace** [comment]
> Any idea why?

Not sure, which version of quarto do you have?

**Comment by Robinlovelace** [comment]
Update on this, after checking here https://github.com/dabreegster/odjitter/tree/qmd it seems that the formatting is not right in the rendered .md doc. Working on it...

**Comment by Robinlovelace** [comment]
Heads-up @dabreegster this is looking better to me, simple can be good!

https://github.com/dabreegster/odjitter/tree/qmd

**Comment by dabreegster** [comment]
> Not sure, which version of quarto do you have?

Ah, I've got 0.3.34, I'll upgrade. Thanks for checking

**Review by dabreegster — APPROVED** [review]
Thank you! I'll play around with other ways to regenerate things like `--help` later, this is an improvement for now and describes the new subpoint flags

**Pull request merged**